frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -167,6 +181,28 @@ const getTopologyEdgeItems = (dc: K8sResourceKind, resources: K8sResourceKind[])
       });
     }
   });
+
+  _.forEach(edgesFromServiceBinding(dc, serviceBindingRequests), (edge) => {
+    // handles multiple edges
+    const targetNode = _.get(
+      _.find(resources, (deployment) => {
+        const name =
+          _.get(deployment, 'metadata.ownerReferences[0].name') || deployment.metadata.name;
+        return name === edge;
+      }),
+      ['metadata', 'uid'],
+    );
+    const uid = _.get(dc, ['metadata', 'uid']);
+    if (targetNode) {
+      edges.push({
+        id: `${uid}_${targetNode}`,
+        type: 'connects-to',
```

Comment:
This should be of type `service-binding`. 
You need to identify the edge type being deleted to call the right delete function.
For now you can contribute the same edge shape inside `shape-providers.ts` for this new type.